tests/datasets/test_geo.py

```diff
@@ -149,10 +149,9 @@ class TestRasterDataset:
     @pytest.fixture(params=[True, False])
     def naip(self, request: SubRequest) -> NAIP:
         root = os.path.join("tests", "data", "naip")
-        crs = CRS.from_epsg(3005)
```

Comment:
It looks like removing this line is causing coverage to drop. We need at least one test where we sample from a CRS different than the file. Maybe we could change the CRS but also change the bounding box to be smaller than `dataset.bounds`? Or use a CRS with the same approximate CRS units as the files?